plexapi/video.py

```diff
@@ -268,25 +281,38 @@ def episodes(self, watched=None):
            Returns:
                 list: of Episode
 
-        
+
         """
         childrenKey = '/library/metadata/%s/children' % self.ratingKey
         return utils.listItems(self.server, childrenKey, watched=watched)
 
-    def episode(self, title):
-        """Find a episode with a matching title.
+    def episode(self, title=None, season=None, episode=None):
```

Comment:
i wasn't sure if I should add season, I only did so episode in show and season accept the same kwargs. I'll remove it